tests/expressions_case/tests.py

```diff
@@ -1,9 +1,14 @@
 from __future__ import unicode_literals
 
+import unittest
```

Comment:
this goes below `from uuid ...` -- whether it's `import` or `from ... import` doesn't affect ordering